src/dbnode/client/session.go

```diff
@@ -3629,7 +3627,7 @@ type enqueueCh struct {
 	metrics              *streamFromPeersMetrics
 }
 
-const enqueueChannelDefaultLen = 32768
+const enqueueChannelDefaultLen = 512
```

Comment:
hmmm? isn't this too small in a sense?

Reply:
Since the enqueue channel just moves things from one incoming channel into requests that go into a host queue, it's never blocked by the network or anything like that so it actually doesn't need a long queue at all (and each item does not require a lot of computation to complete).

I was seeing some non-trivial memory usage in the profiles from this channel since each entry contains data from many replicas and is a very large struct (several hundreds of bytes). So since this was never seeing any lag I think making this small shouldn't hurt anything as it is just shuffling things between other queues and is not blocked by any network/other dependencies.